### PR TITLE
jsonrpc: Pretty-print the json results

### PR DESCRIPTION
**Pull request by cdecker**
Just a small cleanup of the indentation code, so we don't have to reformat all
the issue reports to become readable. This is much closer to what `jq` or
`json_pp` spit out and doesn't have those infinitely long lines.

### CODE REVIEW

**Comment by ZmnSCPxj** [comment]
ACK 578161d7aaad96918b421f68f4df37ace8f16c16

**Comment by rustyrussell** [comment]
Would be nice to see a before-and-after in the commit message for lazy reviewers like me :)

**Comment by cdecker** [comment]
Certainly, here you go.
Before:
```
{ "id" : "037debfd493f2280dd7c0ce54fe66af50403b7f3e123184a0fb439739b16b52ed6", "port" : 9735, "address" : 
	[ 
		{ "type" : "ipv6", "address" : "2a02:aa16:1102:5480:e99:3f63:40a2:83e9", "port" : 9735 } ], "version" : "v0.5.2-2016-11-21-2489-g249b1ac", "blockheight" : 517065, "network" : "bitcoin" }
```

After:
```
{
  "id": "037debfd493f2280dd7c0ce54fe66af50403b7f3e123184a0fb439739b16b52ed6", 
  "port": 9735, 
  "address": [
    {
      "type": "ipv6", 
      "address": "2a02:aa16:1102:5480:e99:3f63:40a2:83e9", 
      "port": 9735
    }
  ], 
  "version": "v0.5.2-2016-11-21-2490-g578161d", 
  "blockheight": 517141, 
  "network": "bitcoin"
}

**Comment by FeedTheWeb** [comment]
that is what it looks like on screen, but it reality it is more like 
`'{\n  "label": "pZtyu9n8Jv", \n  "bolt11": "lntb100u1pdv4qzypp5g5g74uyzxdyfler77twlzuccmjapjnndfnj49m2hmex2c0ng556sdq8v9ekggqcqpxau33dxr0k40sf9mjrtaq85mwzznkf2t94ml0805z8mmxz7g25je3we7z842jwjknc7x06slvfg2y8snhrd33g33vrs6apanex80c0ucqgu6w6d", \n  "payment_hash": "4511eaf08233489fe47ef2ddf17318dcba194e6d4ce552ed57de4cac3e68a535", \n  "msatoshi": 10000000, \n  "status": "paid", \n  "pay_index": 16, \n  "msatoshi_received": 10012300, \n  "paid_timestamp": 1523220556, \n  "paid_at": 1523220556, \n  "expiry_time": 1523224148, \n  "expires_at": 1523224148\n}\n'`
 
which makes it hard to parse

**Comment by cdecker** [comment]
@FeedTheWeb I'm not sure I'm following, this is just as valid JSON as the prior result was, it shouldn't be any harder to parse than previous versions, unless you are using string matching, which would be a hell of a difficult parser to maintain anyway.

**Comment by FeedTheWeb** [comment]
@cdecker  ok, forget what I said.  
I am using python subprocess.Popen() as a wrapper for some functions, which returns that bytestring by default, but it turns out should have used it differently
